CardView: Update demo descriptions (#30108)
DevExpress/DevExtreme: apps/demos/Demos/CardView/ColumnHeaderFilter/description.md

> DevExtreme CardView includes a header filter which allows users to filter cards by field values. Click the filter icon in a field's header panel to open the built-in CardView filter pop-up.

Reviewer: ```suggestion
DevExtreme CardView includes a header filter that allows users to filter cards by field values. Click the filter icon in a field's header panel to open the pop-up CardView filter.
```
